.github/workflows/nydus-config.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Maybe move this file to `misc/nydus-config.json`, and update `README.md` to use this configuration. (it's not a blocker for this PR).